Sources/Moya/MoyaProvider.swift

```diff
@@ -75,8 +75,8 @@ open class MoyaProvider<Target: TargetType>: MoyaProviderType {
     /// of a request for a given `TargetType`.
     public let stubClosure: StubClosure
 
-    /// The manager for the session.
-    public let manager: Manager
+    /// The session for the session.
+    public let session: Session
```

Comment:
> The session for the session

I think this should be changed a bit, e.g. The session for the network session / Moya provider

Reply:
oh damn didn't notice that🤦‍♂ nice catch!

Reply:
Actually I think this doesn't really need a comment, seems self-explanatory so I will remove it for now.